### PR TITLE
Adds police station roofs, new station.

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary

```SUMMARY: Content "Adds police station roofs & a new station. Also includes some small fill_ter changes & increased spawns in general."```

#### Purpose of change
Continues roof project.

#### Describe the solution
I added roofs to police stations and did some minor upgrades.  The original one is now brick to match the others.  Police_2 got a roof gym room and police_1 is new, my minor revisions to the original station got out of hand, and I thought it was better to make it separate.

Spawns are pretty generous to match the new direction in loot availability.  

#### Describe alternatives you've considered
removing the original prison, but doesn't hurt to have 3.

#### Additional context
police & police_roof.  Nothing too special, the brick makes it look much better.
police
police_roof

police_1, police_2ndfloor_1, police_roof_1
has a very large evidence locker and generous armory (@davidpwbrown suggestion) as well as additional cell and police facilities. 
Planning on having an NPC spawn here.  Considered adding secubots to terminal failure but wasn't sure if that was too much.

police_1
police_2ndfloor_1
police_roof_1

police_2 and police_roof_2:
Didn't do much to the ground floor but it got a nice workout room and mini roof to explore.
police_2
police_roof_2
policeL_upper_roof_2

### CODE REVIEW

**Comment by LovamKicsiGazsii** [comment]
The armory is a bit too big in police_1.
It was meant for a max of 6 policemen.
Also, I think you forgot to remove the smaller evidence room (on the top right).

Everything else looks pretty great. I like the brick walls, and the upgrades in general.

File: data/json/mapgen/police_station.json [hunk 1]
```diff
@@ -66,6 +56,160 @@
...
+        "|$^$|..|+-|-+|-+--|-+--|",
+        "|..2|..|.S|S.|t..c|....|",
+        "|.dd|..|.t|t.|...c|Ccc.|",
+        "|.Cd|..|+-|-+|...c|.cc.|",
+        "|..6|..|.&|&.|cccc|.C..|",
+        "|-=-|..----------------|",
+        "|L.L|<.*__,,,,,,,,,,,,,,",
+        "|L.L|<.*__,,,,,,,,,,,,,,",
+        "|L.L|---|_,,,,,,,,,,,,,,",
+        "|L.2FFFF|_,,,,,,,,,,,,,,",
```
Comment (Night-Pryanik) [inline data/json/mapgen/police_station.json]:
```suggestion
        "|L.2FFF.|_,,,,,,,,,,,,,,",
```

File: data/json/mapgen/police_station.json [hunk 2]
```diff
@@ -66,6 +56,160 @@
...
+        "|L.....<|_,,,,,,,,,,,,,,",
+        "|LLLLLLL|_,,,,,,,,,,,,,,",
```
Comment (Night-Pryanik) [inline data/json/mapgen/police_station.json]:
```suggestion
        "|LLLLLL.|_,,,,,,,,,,,,,,",
```

**Comment by Mecares** [comment]
I like your additions maybe one station could have a helipad on the roof if you ever run out of ideas to implement.

**Comment by curstwist** [comment]
> 
> 
> The armory is a bit too big in police_1.
> It was meant for a max of 6 policemen.
> Also, I think you forgot to remove the smaller evidence room (on the top right).
> 
> Everything else looks pretty great. I like the brick walls, and the upgrades in general.

What makes you think it's for 6 police? Just curious.  I can thin it out some.

**Comment by curstwist** [comment]
> 
> 
> I like your additions maybe one station could have a helipad on the roof if you ever run out of ideas to implement.

thanks, I do need to get some helipads going (need to test the wrecks to see if the spawn with that rubble or not first).  The mall will definitely get a helipad, I think these buildings are too small to handle one.

**Pull request merged**